libs/design/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,38 @@
+import { createStyles, TextInput as MantineTextInput } from '@mantine/core';
+
+export interface InputProps {
+  label: string;
+  placeholder: string;
+  description?: string;
+  error?: string;
+  withAsterisk: boolean;
+}
+
+const useStyles = createStyles(theme => ({
+  textInput: {
+    label: {
+      color: theme.colors.fgSubtle[6],
+    },
+  },
+}));
+
+export const TextInput = ({
+  label,
+  placeholder,
+  description,
+  error,
+  withAsterisk,
+}: InputProps) => {
+  const { classes } = useStyles();
+
+  return (
+    <MantineTextInput
+      className={classes.textInput}
+      description={description}
+      placeholder={placeholder}
+      label={`${label}:`}
```

Comment:
Let's not have the colon here, then we can spread the props.

Reply:
All of these custom components were deleted.